fortimanager.json

```diff
@@ -0,0 +1,1193 @@
+                {
+                    "data_path": "action_result.parameter.package",
+                    "data_type": "string",
+                    "example_values": [
+                        "default"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.policy_name",
+                    "data_type": "string",
+                    "example_values": [
+                        "soar-footer-firewall-name"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.policy_type",
+                    "data_type": "string",
+                    "example_values": [
+                        "footer"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.vpn_dst_node",
+                    "data_type": "string",
+                    "example_values": []
+                },
+                {
+                    "data_path": "action_result.data.*.vpn_src_node",
+                    "data_type": "string",
+                    "example_values": []
+                },
+                {
+                    "data_path": "action_result.parameter.adom",
+                    "data_type": "string",
+                    "example_values": [
+                        "root"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.total_firewall_policies",
+                    "data_type": "numeric",
+                    "example_values": [
+                        6
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        }
+    ],
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "certifi",
+                "input_file": "wheels/py3/certifi-2023.7.22-py3-none-any.whl"
+            },
+            {
+                "module": "charset_normalizer",
+                "input_file": "wheels/py3/charset_normalizer-2.0.12-py3-none-any.whl"
+            },
+            {
+                "module": "idna",
+                "input_file": "wheels/py3/idna-3.4-py3-none-any.whl"
+            },
+            {
+                "module": "pyfmg",
+                "input_file": "wheels/py3/pyfmg-0.8.5.4-py3-none-any.whl"
+            },
+            {
+                "module": "requests",
```

Comment:
you have to delete the requests module from dependencies and the wheel files after pre-commit runs, then push - limitation of our current setup